.vscode/extensions.json

```diff
@@ -2,6 +2,7 @@
 	// See http://go.microsoft.com/fwlink/?LinkId=827846
 	// for the documentation about the extensions.json format
 	"recommendations": [
-		"ms-vscode.vscode-typescript-tslint-plugin"
+		"dbaeumer.vscode-eslint",
+		"eamodio.tsl-problem-matcher"
```

Comment:
any compelling reason why eslint replaces tslint? We now need 2 extensions instead of 1

Reply:
These are the default extension recommendations that are present when creating a new extension. There is no compelling reason to replace eslint replaces tslint. We can go back to vscode-typescript-tslint-plugin" if you suggest.

Reply:
TSLint has been deprecated. https://github.com/palantir/tslint/issues/4534.

There is a WIP analytics lsp PR which switches to ESLint: https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/161/files

Reply:
ack